core/chains/evm/config/toml/defaults.go

```diff
@@ -33,48 +32,68 @@ var (
 )
 
 func init() {
-	// read the defaults first
+	// read all default configs
+	initReadDefaults()
+
+	// check for and apply any overrides
+	initApplyEVMOverrides()
```

Comment:
I don't think this indirection is adding value. Please consider setting the global vars from this func, one way or another.
https://github.com/smartcontractkit/chainlink/pull/15617/files#r1883872492